src/Search_Replace_Command.php

```diff
@@ -328,7 +328,7 @@ private function php_handle_col( $col, $primary_keys, $table, $old, $new ) {
 		$replacer = new \WP_CLI\SearchReplacer( $old, $new, $this->recurse_objects, $this->regex, $this->regex_flags );
 
 		$where = $this->regex ? '' : " WHERE `$col`" . $wpdb->prepare( ' LIKE %s', '%' . self::esc_like( $old ) . '%' );
-		$primary_keys_sql = esc_sql( implode( ',', $primary_keys ) );
+		$primary_keys_sql = '`' . esc_sql( implode( '`,`', $primary_keys ) ) . '`';
```

Comment:
I'm not 100% sure, but I think this is vulnerable to SQL injection.

From the [docs for `esc_sql()` function](https://codex.wordpress.org/Function_Reference/esc_sql):
```
Note: Be careful to use this function correctly. It will only escape values to be used in strings in the query. That is, it only provides escaping for values that will be within quotes in the SQL (as in field = '{$escaped_value}'). If your value is not going to be within quotes, your code will still be vulnerable to SQL injection. For example, this is vulnerable, because the escaped value is not surrounded by quotes in the SQL query: ORDER BY {$escaped_value}. As such, this function does not escape unquoted numeric values, field names, or SQL keywords.
```